extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson-common/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/ResteasyReactiveJacksonProviderDefinedBuildItem.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.resteasy.reactive.jackson.deployment.processor;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public class ResteasyReactiveJacksonProviderDefinedBuildItem extends MultiBuildItem {
```

Comment:
I think we should always add javadoc to our build items.

Reply:
done